README.md

```diff
@@ -18,6 +18,44 @@ MySQL and PHP are not prerequisites.  Although the eventual target system is Mag
 database, the actual target for the migration scripts is a Magento2-compatible CSV file, not the database.  Magento2
 itself requires PHP, and handles the task of importing from CSV into MySQL.
 
+## Setting Up
+
+Once this repo is cloned and prerequisites are installed, all that is required is to install the local node package
+dependencies using the standard command:
```

Comment:
You mean "and prerequisites are installed"?  (The previous heading being "Prerequisites")